data/json/recipes/recipe_deconstruction.json

```diff
@@ -2837,6 +3010,21 @@
     "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "WRENCH", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
     "components": [ [ [ "motor_tiny", 3 ] ], [ [ "processor", 3 ] ], [ [ "RAM", 2 ] ], [ [ "plastic_chunk", 2 ] ] ]
   },
+  {
+    "result": "sewing_kit",
+    "type": "uncraft",
+    "time": "30 s",
+    "components": [ [ [ "scrap", 2 ] ], [ [ "plastic_chunk", 1 ] ] ]
```

Comment:
I think this should yield a needle.

Reply:
I did the disassemble yield the needed crafting components.

Reply:
We could change the sewing kit recipe to allow the needle OR the 2 metal scraps, but as the tailor's kit needs both the sewing kit and the curved needle I thought it would be strange as it seems that the needle and the sewing kit are quite are different things.

Reply:
No, actually, what you have there is fine. I didn't realise that you could make a sewing kit without a needle.